moleculekit/tools/train_xgboost.py

```diff
@@ -0,0 +1,42 @@
+from moleculekit.smallmol.smallmollib import SmallMolLib
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import train_test_split
+from xgboost import XGBRegressor
```

Comment:
move the imports inside the function, otherwise moleculekit will crash if the user doesn't have xgboost or sklearn installed and many users might not need them